src/index.html

```diff
@@ -1,13 +1,324 @@
+      <div class="header__content">
+        <div class="header__top top-bar">
+          <a href="#">
+            <img
+            class="top-bar__logo"
+            src="images/logo-main.svg"
+            alt="MyBike Logo">
+          </a>
+
+          <div class="top-bar__icons">
+            <a href="#" class="icon icon--phone"></a>
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+        </div>
+
+        <div class="header__buttom">
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__top top-bar">
+          <a href="#home" class="top-bar__logo-link">
+            <img
+            class="top-bar__logo"
+            src="images/logo-menu.svg"
+            alt="MyBike Logo">
+          </a>
+
+          <a href="#" class="icon icon--close"></a>
+        </div>
+
+        <div class="menu__bottom">
+          <div class="menu__bottom">
+            <nav class="menu__nav nav">
+              <ul class="nav__list">
+                <li class="nav__item">
+                  <a href="#home" class="nav__link">Home</a>
+                </li>
+
+                <li class="nav__item">
+                  <a href="#move-free" class="nav__link">About Us</a>
+                </li>
+
+                <li class="nav__item">
+                  <a href="#compare" class="nav__link">Compare Bikes</a>
+                </li>
+
+                <li class="nav__item">
+                  <a href="#details" class="nav__link">Details</a>
+                </li>
+
+                <li class="nav__item">
+                  <a href="#contact" class="nav__link">Contacts</a>
+                </li>
+              </ul>
+            </nav>
+
+            <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+              +1 234 5555-55-55
+            </a>
+
+            <a href="tel:+1 234 5555-55-55" class="menu__book-a-test-ride">
+              Book a test ride
+            </a>
+          </div>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section class="move-free" id="move-free">
+            <h2 class="move-free__title section-title">Move Free</h2>
+            <p class="move-free__content">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+            </p>
+          </section>
+          <section class="compare" id="compare">
+            <h2 class="section-title">Compare Bikes</h2>
+            <div class="compare__bikes">
+              <article class="product compare__bike">
+                <a href="#home" class="product__link">
+                  <img
+                    src="images/product/sporty.png"
+                    alt="Sporty 4"
```

Comment:
It is better to give more meaningful `alt` descriptions, for example `Bike model Sporty 4`.
Fix this in all code.

Reply:
done